spec/requests/provider_suggestions_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+describe "/provider-suggestions", type: :request do
+  context "when provider suggestion is blank" do
+    it "returns bad request (400)" do
+      get "/provider-suggestions"
+
+      expect(response.status).to eq(400)
+      expect(JSON.parse(response.body)).to eq("error" => "Bad request")
+    end
+  end
+  context "when provider suggestion is less than three characters" do
+    it "returns bad request (400)" do
+      get "/provider-suggestions?query=St"
+
+      expect(response.status).to eq(400)
+      expect(JSON.parse(response.body)).to eq("error" => "Bad request")
+    end
+  end
+  context "when provider suggestion ids valid" do
```

Comment:
Could we add a test in here to make sure the stubbed request is made? :eyes:

Reply:
(And the subsequent handling of the response)

Reply:
👍